app/actions/user.js

```diff
@@ -43,7 +55,7 @@ export const auth0Login = (dispatch) => {
 
         // creates the user after successful login by passing the credentials having accessToken, idToken
         // and refreshToken to store in DB
-        createOrModifyUserToken(credentials);
+        createOrModifyUserToken({ ...credentials, expirationTime });
```

Comment:
I do not think it would have been necessary to store the expiration time explicitly and extend the database scheme for it, as it can always be retrieved from the `accessToken` with `getExpirationTimeStamp(accessToken)` again, doesn't it?

Reply:
@ankitecd please tell me if I miss anything here with demands storing the expirationTime in an extra field of the scheme.

Reply:
Well I thought it would decode the token each time it is going to make a request and we are going to fetch for access token from the schema itself most of the time, so thought to include in the schema itself and it will add the expiration time while creating itself. Let me know if the implementation needs to be changed.

Reply:
Let's keep it for now.